src/zenml/cli/stack_recipes.py

```diff
@@ -0,0 +1,806 @@
+
+    Args:
+        git_stack_recipes_handler: The GitStackRecipesHandler instance.
+        stack_recipe_name: The name of the stack_recipe.
+        force: Force the redownload of the stack_recipes folder to the ZenML config
+            folder.
+        path: The path at which you want to install the stack_recipe(s).
+    """
+    git_stack_recipes_handler.pull(branch="main", force=force)
+
+    stack_recipes_dir = os.path.join(os.getcwd(), path)
+    io_utils.create_dir_if_not_exists(stack_recipes_dir)
+    try:
+        stack_recipes = git_stack_recipes_handler.get_stack_recipes(
+            stack_recipe_name
+        )
+    except KeyError as e:
+        cli_utils.error(str(e))
+
+    else:
+        for stack_recipe in stack_recipes:
+            destination_dir = os.path.join(os.getcwd(), path, stack_recipe.name)
+            if LocalStackRecipe(
+                name=stack_recipe.name, path=Path(destination_dir)
+            ).is_present():
+                if force or cli_utils.confirmation(
+                    f"Stack recipe {stack_recipe.name} is already pulled. "
+                    "Do you wish to overwrite the directory at "
+                    f"{destination_dir}?"
+                ):
+                    fileio.rmtree(destination_dir)
+                else:
+                    cli_utils.warning(
+                        f"Stack recipe {stack_recipe.name} not overwritten."
+                    )
+                    continue
+
+            cli_utils.declare(f"Pulling stack recipe {stack_recipe.name}...")
+
+            io_utils.create_dir_if_not_exists(destination_dir)
+            git_stack_recipes_handler.copy_stack_recipe(
+                stack_recipe, destination_dir
+            )
+            cli_utils.declare(
+                f"Stack recipe pulled in directory: {destination_dir}"
+            )
+            track_event(
+                AnalyticsEvent.PULL_STACK_RECIPE,
+                {"stack_recipe_name": stack_recipe.name},
+            )
+
+
+@stack_recipe.command(
+    help="Run the stack_recipe that you previously pulled with "
+    "`zenml stack recipe pull`"
+)
+@click.argument("stack_recipe_name", required=True)
+@click.option(
+    "--path",
+    "-p",
+    type=click.STRING,
+    default="zenml_stack_recipes",
+    help="Relative path at which you want to install the stack_recipe(s)",
+)
+@click.option(
+    "--yes",
+    "-y",
+    "force",
+    is_flag=True,
+    help="Force the run of the stack_recipe. This deletes the .zen folder from the "
+    "stack_recipe folder and force installs all necessary integration "
+    "requirements.",
+)
+@click.option(
+    "--shell-executable",
+    "-x",
+    type=click.Path(exists=True),
+    required=False,
+    envvar=SHELL_EXECUTABLE,
+    help="Manually specify the path to the executable that runs .sh files. "
+    "Can be helpful for compatibility with Windows or minimal linux "
+    "distros without bash.",
+)
+@pass_git_stack_recipes_handler
```

Comment:
An option to automatically import the stack would be nice here, maybe something like `zenml stack recipe deploy <RECIPE_NAME> --import` (Or even let users specify an import name)

Reply:
Hamza mentioned that the import should be the intended behaviour so I had made that the default and added `--no-import` as an option instead.